src/client/components/App/index.js

```diff
@@ -50,19 +68,24 @@ const App = () => {
         isUserAdmin={isUserAdmin}
         isUserTeacher={isUserTeacher}
         idToken={idToken}
+        setError={setError}
       />
+      {error && <ErrorAlert err={error.err} msg={error.msg} />}
```

Comment:
@kilometerskang Curious if there's any way to avoid all the setError propagation up the component tree. Did u look into this? https://reactjs.org/docs/error-boundaries.html

Reply:
thanks, let me look